src/common/config/currency-converter-options.ts

```diff
@@ -1,9 +1,11 @@
 export interface CurrencyConverterOptions {
     precision: number;
     isEnabled: boolean;
+    separator:string;
```

Comment:
Here is a missing space after the colon.

Reply:
I think this variable name can be optimized. `separator` is a little bit too generic. What does it separate?

Reply:
would `numberSeparator` be a better variable name?

Reply:
What does it separate?